### ISSUE TITLE
Create a Bitcoin Design card game

### ISSUE THREAD

**Issue by GBKS** [BitcoinDesign/Meta#523]
![bitcoin-design-card-game-cover-2](https://github.com/BitcoinDesign/Meta/assets/695901/6add98b4-7088-40b5-b159-5b11145de13f)

This idea came up during planning for [BTC Prague](https://github.com/BitcoinDesign/Meta/issues/495), to have a casual and fun way to introduce hack-day and expo attendees to what this community is about. During hack-day, there could be a table set aside for playing this game, and at the expo booth it might be a bit of eye catcher and conversation starter.

Game requirements are:
- Super easy to learn
- A round takes just a few minutes
- Highly collaborative
- Capture important aspects of the community

I put together a starting point for an initial design ([Figma](https://www.figma.com/file/gjHBrxfmJDMa3PN4X1gzpc/Bitcoin-Design-Card-Game?node-id=0%3A1&t=JA17DMSA3dDUX4WC-1)) and a video to explain the concept.

📀[Watch the video](https://youtu.be/hLx5GezHVjw)📀

We would need about 90 cards in total:
- 24 problem cards
- 45 solution cards
- 15 special effect cards
- 6 goal cards

Trickiest part is to come up with enough ideas for cards, and I think community participation is essential. Everyone has some experience and insight into bitcoin and design projects that could flow into this. If you have ideas, please share them here in a reply.

From a quick check, it would cost about 25€ to print a custom card deck.

What do you think?

**Comment by aassoiants** [comment]
Just popping in to say you're brilliant

**Comment by GBKS** [comment]
Latest status on the cards. I switched away from 1960's sci-fi magazine style because the AI doesn't have enough material to create diverse images from. I tested vinyl designer toys and think it works well. You can create lots of compact characters oozing with personality that are easy to recognize and describe during the game.

Below are 24 cards. We need a total of 90, so still quite a bit of stuff to figure out. If you want to help, chipping in with card ideas would be amazing. Do you have an experience with design/bitcoin/open-source that can be bundled into a card, or card combo? Leave a note.

![cards-vinyls](https://github.com/BitcoinDesign/Meta/assets/695901/e325d30e-34bd-4bcc-9404-081c31df1a0a)

**Comment by mouxdesign** [comment]
Adding in some ideas, some of the ideas a bit fun and playful and some are a bit more process orientated.

***(The text itself would require some refinement as they are a bit wordy, also some of them you've already included)

The solutions are a mixture between UX research solutions straight from the UX Research toolkit and then also UI solutions
## Solution card ideas
1. User flow
2. Design Sprint
3. Competitive research
4. UX Research repository
5. Design System 
6. Usability Testing
7. User journey mapping
8. Jobs to be done
9. Build in public
10. Rapid prototyping
11.  Ask the community
12. UX Research plan
13. Affinity mapping
14. Crazy 8 sketching
15. Heuristic evaluation
16. Co-ordinate with devs
17. Create a brand
18. User guides/FAQS
19. User story
20. Guerilla testing
21. Seek collaborators
22. UX Strategy
23. Work on copy
24. Work on onboarding flow
25. Work on send screen
26. Work on receive screen
27. Design information architecture
28. Create a contact option
29. Create milestones
30. Setup github project
31. Add prompt reminders
32. Add more contrast to design
33. Test error states
34. Design error states
35. Bucket user feedback
36. Create call to action
37. Create visual hierarchy
38. Add more white space
39. Provide more feedback on the interface
40. Generative interviews
41. Progress indicators
42. Split to 2 screens/reduce info on the screen
43. Allow personalization
44. Add more visuals

## Problems
1. Accessibility
2. We are getting a lot of customer emails with questions (image idea inbox overload) (confused users)
3. Joker (this would be the a sort of joker card, it's the problem that's faced by that unique team so allows the players to tackle the problem they are facing right at that moment, they might scribble this on a post it and stick it to this card)

## Special effects
1. AI art
2. Ask Chat GPT
3. Throw the bucket (we really have no idea that the solution might be...humor)
5. Spin it (spin a bottle the person who it lands on has to tackle the problem)
7. Grab beer/wand
8. Get a product wizard (product manager)
9. Turn on some music
10. Ship it
11. 30 seconds (person has 30 seconds to react)- clock image with it ticking away
12. Double trouble (you get 2 problem cards instead of 1)
13. Collab dance (collaborate with the person on your right for 5 mins)
14. Ask a question
15. Problem stack (the person on your left gets to add on one more UX problem to the one you received)

**Comment by aassoiants** [comment]
A few humble additions:

**Special Events**
1. Project got funded
2. New contributor joined the project
3. New BIP released

**Problem cards**
1. User doesn't understand lightning
2. Team doesn't ship quickly enough
3. Project lead losing interest

**Solution cards**
1. Run team facilitation session
2. Align on product vision
3. Go to local bitcoin meetup

Quick note that problem card and solution card categories can be replicated to stir creativity. For example, what would a Tech solution card look like? Or a Events category for Problem cards?

**Comment by mouxdesign** [comment]
Generated some images for the card game here. Just linking to the figma file which is 100% open. Will generate more images tomorrow as well.

[Figma file.](https://www.figma.com/file/Iu35jdOWxbooORdnCCmFx9/Card-Game-Images?type=design&t=TmaHaACPmFaF16Ow-1)

**Comment by GBKS** [comment]
![card-game-progress-23-05-15](https://github.com/BitcoinDesign/Meta/assets/695901/b55906e7-0462-463d-8512-0a604a225f9f)

Alright, we've got 90 cards. I will order 6 of these tomorrow morning. Will cost around $20 each (+around $50 shipping to get it in time). I'm ordering them via https://www.makeplayingcards.com in skat size (59x91mm) on standard smooth card stock, full-color print, each deck in a plain white tuck box. Nicer boxes with custom prints and instruction booklets are possible, but increase the price by over 50%, so keeping it simple here for this first version (if people like this format, we can keep working on it). If you want your own copy, let me know today, but you have to attend BTC Prague so I can give it to you in person.

**Comment by GBKS** [comment]
The deck is sent off for printing. Aaaaand I already discovered a problem card with the wrong solution image and name. Oh well. Here's an image of test prints I made with my cheap home printer on cheap printer paper. It did help get a sense for the size, especially to check whether those small images on problem cards are easily recognizable.

![carg-game-test-prints-1600](https://github.com/BitcoinDesign/Meta/assets/695901/fdb66939-99e8-4d58-86e8-7c0a52f9a894)

I should get the decks delivered in 6-9 biz days.

**Comment by GBKS** [comment]
And here's the real deal. Quality is decent, was hoping the print would be a little better (but it's still experimental and we chose a cheaper option, so all good there). Quite a thick stack of cards. Now is it fun to play or a good conversation starter? We'll find out.

![image](https://github.com/BitcoinDesign/Meta/assets/695901/ee127f97-8685-4364-bdb7-216e69f803a1)

**Comment by GBKS** [comment]
Just a quick update now with some thoughts coming out of BTC Prague. Overall, the card game was a hit. Everyone who played it had fun (at least that's what it looked like). People got it right away with minimal explanation, we didn't even need to play several rounds for players to pick up how it works. Several people asked if we sell the cards. I saw some go through the deck and look through all the images, even picking out their favorites. I also got a bunch of questions about how they were generated. We gave away all the card decks (we had just 10) to organizers and long-term friends of the community.

I'd like to collect more feedback, but here are some possible things to do to continue with this project:

- Mock up and spec out 2-3 different deck variations at different price points and levels of quality. The test decks cost €18 each + €50 total in shipping. With a [cardboard box](https://www.makeplayingcards.com/promotional/card-game-boxes.html) and [instruction manual](https://www.makeplayingcards.com/promotional/custom-game-booklets.html) this could go up to €40+. But this is strongly based on quantity. If you order 100+, price goes back down by 50% or more. Could be even more with specials like [holographic cards](https://www.makeplayingcards.com/custom-hologram-cards.aspx). What combo of these things hit the sweet spot? I think we need to spec out a few options.
- Crowdsource more cards and visuals. Maybe more people want to contribute now that they have seen it in action? 
- Ideate expansion packs? A smaller extra deck focused on technical aspects or bitcoin culture? Maybe with unique rules?
- Open-source the images, Figma file, and everything else and create a nice landing page for the project
- Create other products with the images. We printed some as pillows for the design lounge and were also asked about whether we sell them
- Sell decks via a Nostr store
- Start a [Geyser](https://geyser.fund) funding campaign.
- Do nothing (also an option)

And lastly, I used some of the images and cards for presentations and the made for a really nice asset library.

<img width="754" alt="image" src="https://github.com/BitcoinDesign/Meta/assets/695901/25b6e7e6-81f6-4105-9392-fa8735ff8011">

That's it for now. What do you think? What would you like to see happen?

**Comment by moneyball** [comment]
Thoughts on selling the card game and using proceeds to fund the Bitcoin Design Community and future grants? If marketed as a donation toward FOSS bitcoin, perhaps bitcoiners wouldn't balk at a price of like $49? Selling 10,000 would be half a million raised for grants!

**Comment by GBKS** [comment]
Love the idea and ambition. A crowd-funding campaign might be ideal for this with several milestones and donation tiers. Someone could get the basic deck for a lower price, or get the holographic one with the extra poster for a higher price. Having a cheaper deck that we sell many of might even be financially smart since the unit price goes down so much with higher volume orders. Either way, we'd really have to step things up to achieve this, but could very well be worth it. I'll try to put together some possible directions and then get more feedback.

**Comment by GBKS** [comment]
Alright, so we decided to do another edition of the deck for this year's [BTC Prague](https://github.com/BitcoinDesign/Meta/issues/637).

Plan is to print another 50 decks of 90 cards, with a booklet and a rigid (cardboard) box (last time we only had a plain white paper box). Print cost for 10 would be €37.54 each, and for 50 it's only €18.40 each. Wild how the cost drops with larger quantities (at 10,000 it's €3.29 each and shipping cost will likely be your main cost for distribution at that point). Either way, 50 is a small enough amount that we won't be stuck with them if there's no interest, and we can see how it works out with a better production value.

So we have a few things to do:

- Revise the cards - crowdsource some more ideas
- Design the box and booklet
- Prep the production materials and triple-check everything
- Order them

About 5.5 weeks until the conference, so we really want to have the order out in 2 weeks (max 3).

Let's start with step 1 and ideate. I'll put out a call for ideas in the Discord.

**Comment by GBKS** [comment]
The second edition is off to print. Here are the final graphics:

![2024-cards](https://github.com/BitcoinDesign/Meta/assets/695901/7d3eff28-1a8e-4a3d-8c4a-4a9438e61bab)

![card-booklet-2](https://github.com/BitcoinDesign/Meta/assets/695901/1d6aeece-277b-491f-bc6b-28e690fe59b3)

![box-mock](https://github.com/BitcoinDesign/Meta/assets/695901/eecfc96f-2673-40c5-8f44-fd0ac00b1a33)

8 cards were replaced, the others are the same. New are the booklet and box. Fingers cross everything turns out well.

**Comment by GBKS** [comment]
I received the decks and they look great. It will be really nice to have them for the conference.

![IMG_7532](https://github.com/BitcoinDesign/Meta/assets/695901/d4f81585-2ccb-45b5-8856-2f3c0a6b31ae)

**Comment by GBKS** [comment]
We did not get to play the game much in [Prague this year](https://bitcoindesign.substack.com/p/bitcoin-design-newsletter-61). During the time we wanted to do that, the room was empty as everyone headed out to another panel stacked with big names. There are some card decks left, reach out if you'd like one. I'll close this issue for now, we can create a new one for any future updates we may want to do.
